example-app/SantokuApp/src/screens/demo/cache/CacheScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import {DemoStackParamList} from 'navigation/types';
+import React, {useCallback, useEffect, useState} from 'react';
+import {ScrollView, StyleSheet, View, RefreshControl} from 'react-native';
+import {Text} from 'react-native-elements';
+
+import {Button} from '../../../components/button/Button';
+import {FileInfo} from './FileInfo';
+import {useCacheDirectory} from './useCacheDirectory';
+
+const ScreenName = 'Cache';
+const Screen: React.FC = () => {
+  const {topLevelFileInfos, reloadCacheDirectoryItemsAsync, clearCacheDir} = useCacheDirectory();
+  const [refreshing, setRefreshing] = useState<boolean>(false);
+
+  const onRefresh = useCallback(() => {
+    setRefreshing(true);
+    reloadCacheDirectoryItemsAsync()
+      .catch(e => {
+        console.log(e);
+      })
+      .finally(() => {
+        setRefreshing(false);
+      });
+  }, [reloadCacheDirectoryItemsAsync]);
+
+  useEffect(() => {
+    reloadCacheDirectoryItemsAsync().catch(e => console.log(e));
+  }, [reloadCacheDirectoryItemsAsync]);
```

Comment:
`onRefresh`で処理を纏めてもいいかなと感じました。

```suggestion
  useEffect(() => {
    onRefresh();
  }, [onRefresh]);
```